common/helix_client.cpp

```diff
@@ -26,7 +26,7 @@
 
 #include "common/network_util.h"
 
-DECLARE_int32(port);
+DEFINE_int32(port, 9090, "Port of the server");
```

Comment:
why do we need this change?

Reply:
In general, the refactor is because of the new dependency of rocksdb_replicator on helix_client. Since rocksdb_admin already depends on rockdb_replicator, it would've created a circular dependency without this refactor. So I moved helix_client to common.
In particular, I had to define the PORT here since this is the common file, all other files which need to use this flag will include this common file.